fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneIndexMaintenanceTest.java

```diff
@@ -781,6 +784,85 @@ public void ensureValid() throws IOException {
         }
     }
 
+    @Test
+    void concurrentUpdate() throws IOException {
```

Comment:
Just to make sure I understand: This test ensures that concurrent updates in the same transaction do not crash, hang or corrupt the index, right?

Reply:
I will add that as a comment.